data/json/items/gun/84x246mm.json

```diff
@@ -7,11 +7,12 @@
     "symbol": "(",
     "color": "green",
     "name": { "str": "M3 recoilless rifle" },
-    "description": "Manufactured in Sweden, the Carl Gustaf M3 is a breech-loading 84 millimeter man-portable, reusable, multi-role recoilless rifle commonly used by the US military.",
+    "description": "Manufactured in Sweden, the Carl Gustaf M3 is a breech-loading 84 millimeter man-portable, reusable, multi-role recoilless rifle commonly used by the US military.  Under normal circumstances, users should refrain from shooting more than 6 rounds per day to avoid undue risk of brain injury...",
```

Comment:
```suggestion
    "description": "Manufactured in Sweden, the Carl Gustaf M3 is a breech-loading 84 millimeter man-portable, reusable, multi-role recoilless rifle commonly used by the US military.  Under normal circumstances, users should refrain from shooting more than 6 rounds per day to avoid undue risk of brain injury…",
```